src/script/api/script_story_page.cpp

```diff
@@ -218,10 +218,57 @@ static inline bool StoryPageElementTypeRequiresText(StoryPageElementType type)
 	return ScriptObject::Command<CMD_REMOVE_STORY_PAGE_ELEMENT>::Do(story_page_element_id);
 }
 
+/* static */ bool ScriptStoryPage::IsValidStoryPageButtonColour(StoryPageButtonColour colour)
```

Comment:
story.cpp has a `IsValidColours()`

Reply:
Those are using non-API enumerated constants, I thought it shouldn't be using them.